### DISCUSSION TITLE
(Feature request) Support to exclude private repositories in wakatime plugin

### DISCUSSION THREAD

**Discussion by jopemachine** [lowlighter/metrics#1048] [Ideas]
Wakatime plugin looks very useful, but I think it would be great for me to hide my github private repositories.

Wakatime supports deleting projects in its own Dashboard, but it is manual, not support excluding all private repositories.

I would be happy if I can exclude only private repositories in the Wakatime plugin (through a related option).

Many thanks for maintaining this awesome library.

**Comment by lowlighter** [comment]
I think we could add a visibility settings for this
Thanks for your feedback 👍

**Reply by lowlighter** [reply]
Was shipped in #1052
